src/mixins/embed.js

```diff
@@ -323,11 +323,13 @@ export default {
                 isSubscribed,
                 isTrialSubscriber,
                 serverCookieSupported,
+                civicLikerVersion,
               } = myData;
               this.isLoggedIn = !!liker;
               this.isCreator = liker === this.id;
               this.isSubscribed = isSubscribed;
               this.isTrialSubscriber = isTrialSubscriber;
+              this.civicLikerVersion = civicLikerVersion;
```

Comment:
Should first defined `civicLikerVersion` in `data()` before using it as `this.civicLikerVersion`